.happy/terraform/envs/dev/main.tf

```diff
@@ -6,8 +6,11 @@ module "stack" {
   deployment_stage = "dev"
   stack_prefix     = "/${var.stack_name}"
   k8s_namespace    = "czid-dev-happy-happy-env"
+  app_name         = var.app
   additional_env_vars = {
     API_URL = "https://sandbox.czid.org"
+    NEXTGEN_ENTITIES_URL = "http://ryan-test-entities.czid-dev-happy-happy-env.svc.cluster.local:8008"
```

Comment:
Have you poked the Happy team about how to update these values on the fly? I think `happy config` is one possible option, but it would be nice to be able to launch different remote-dev stacks with different configurations.

Reply:
I guess I never changed this for dev, but for other envs (sandbox, staging prod), I've been putting ENV vars into Github.  I think it's probably fine for dev to leave it here though, since we generally won't be deploying that  via Github.